public/index.html

```diff
@@ -101,6 +132,37 @@ <h2>PayJoin here to open these channels</h2>
 				channels.removeChild(channels.children[channels.children.length - 2]);
 			}
 
+			async function get_recommended_channels(){
+				document.getElementsByClassName('recommendation-button-wrapper')[0].classList.add('invisible')
+				try {
+					recommendations = await (await fetch('/recommended', {
+						method: 'GET',
+					})).json()
+					document.getElementsByClassName('routing-node-alias')[0].innerHTML = recommendations.routing_node.alias
+					document.getElementsByClassName('routing-node-capacity')[0].innerHTML = recommendations.routing_node.capacity
+					document.getElementsByClassName('routing-node-count')[0].innerHTML = recommendations.routing_node.active_channel_count
+					document.getElementsByClassName('edge-node-alias')[0].innerHTML = recommendations.edge_node.alias
+					document.getElementsByClassName('edge-node-capacity')[0].innerHTML = recommendations.edge_node.capacity
+					document.getElementsByClassName('edge-node-count')[0].innerHTML = recommendations.edge_node.active_channel_count
+					Array.from(document.getElementsByClassName('recommendation-node-wrapper')).forEach(el => el.classList.remove('invisible'));
+				} catch (error) {
+					console.error(error)
+					alert("Something went wrong. Please try again.")
+				}
+			}
+
+			function add_recommended_node(type = 'routing_node') {
+				const node = recommendations[type];
+				if (!node) {
+					alert("Something went wrong. Please try again.");
```

Comment:
Can we explain what went wrong and the best recourse for someone seeing the error

Reply:
Maybe just something like "Can't connect to node sources right now."